src/structures/Webhook.js

```diff
@@ -213,6 +213,15 @@ class Webhook {
     return this.client.api.webhooks(this.id, this.token).delete({ reason });
   }
 
+  /**
+   * The url of this webhook
+   * @type {string}
+   * @readonly
+   */
+  get url() {
+    return this.client.options.http.api + this.client.api.webhooks[this.id][this.token];
```

Comment:
apologies, just noticed, this is a bit better looking: `... .webhooks(this.id, this.token)` and we use this in other places

Reply:
no worries